### DISCUSSION TITLE
Nighly Versions - 2.12.x to 2.99.x

### DISCUSSION THREAD

**Discussion by Nazul** [pnp/powershell#4531] [Q&A] [answered]
After some nightly releases under 2.12.x versions, the releases started to be listed as 2.99.x. I looked for a version roadmap or anything that explains why the number changed this way, and particularly, if there is an ETA for a new stable release.

Is this something that can be found anywhere?

**Comment by gautamdsheth** [comment]
hi @Nazul , this is part of the upcoming 3.x release which will be based on .NET 8 and PS 7.4
It removes support for .NET 6 and PS 7.2 , so you will need to execute the cmdlets in PS 7.4 or later versions

Some more details about it:

https://github.com/pnp/powershell/blob/dev/MIGRATE-2.0-to-3.0.md

**Reply by Nazul** [reply]
Thanks for explaining it. Isn't an ETA or timeline/roadmap for this new version?

**Reply by gautamdsheth** [reply, marked as answer]
We dont have an ETA right now , maybe in 30-45 days or maybe earlier. There are breaking changes in this release and we appreciate any feedback related to the usage of these nightly builds

**Reply by Nazul** [reply]
Thanks for the quick response
